jwst/white_light/white_light_step.py

```diff
@@ -2,6 +2,7 @@
 
 from ..stpipe import Step, cmdline
 from .. import datamodels
+from ..source_catalog.source_catalog import _replace_suffix_ext
```

Comment:
I'm not a big fan of using semi-generic routines like this that come from another step, but it works for now. If we really need this kind of capability, we should put it into a lib routine. @stscieisenhamer may know if we've already got something like this somewhere in stpipe or datamodels.